src/lib/util/RandomUtils.ts

```diff
@@ -0,0 +1,5 @@
+export function generateRandomNumber(): string {
```

Comment:
I think it's strange that generateRandomNumber returns a string, I'd expect a number.